ols/app/endpoints/ols.py

```
@@ -383,22 +383,22 @@ def check_tokens_available(
     except psycopg2.Error as pg_error:
         message = "Error communicating with quota database backend"
-        logger.error(message)
+        logger.error("%s: %s", message, pg_error)
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail={
                 "response": message,
-                "cause": str(pg_error),
+                "cause": "Database connection error",
             },
         )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)